src/server/rpc/procedures/iex-trading/iex-trading.js

```diff
@@ -94,6 +94,35 @@ StockConsumer.dailyPercentChange = function(companySymbol) {
         });
 };
 
+/**
+ * Get historical closing price information about the specified company
+ * @param {String} companySymbol Company stock ticker symbol
+ * @param {String} range Time period (e.g. 1m, 3m, 1y, etc) to retrieve prices for
+ * @returns {Object} Historical price information
```

Comment:
Object should probably be `Array` instead since it is a list of points when received by the netsblox client

Reply:
This is changed in 476149b